sofa-mesh/zh_CN/SUMMARY.md

```diff
@@ -1,26 +1,26 @@
-- [介绍](./Home)
+- [介绍](./Home.md)
```

Comment:
`.Home` this is a doc URL not the absolute path of the doc file, so the suffix of file format `.md` is not required.